fuel-vm/src/interpreter/internal/message_tests.rs

```diff
@@ -2,6 +2,7 @@ use crate::interpreter::memory::Memory;
 
 use super::*;
 use fuel_tx::Create;
+use fuel_types::bytes::SerializableVec;
```

Comment:
Now `SerializableVec` is no longer found in `super::*`, so I must explicitly add it here.